src/components/Cars/Cars.jsx

```diff
@@ -80,7 +80,7 @@ const Cars = () => {
                   </p>
                   <div className="mt-4 flex justify-center gap-4 text-gray-400">
                     <Link
-                      to="/"
+                      to="/car"
```

Comment:
Kindly update this path to include the id of the car in the URL params
```suggestion
                      to={`/cars/${car.id}`}
```